internal/rdma/rdma-test.go

```diff
@@ -224,3 +228,95 @@ func ValidateRDMAResults(results map[string]string) (bool, error) {
 	// If everything is valid
 	return true, nil
 }
+
+// DeleteMofedDeiverNode delete mofed inventory.
+func DeleteMofedDeiverNode(clientset *clients.Settings, podName, namespace, nodeName string) (string, error) {
```

Comment:
To make more use of this function, if we could add an argument to pass the command we want to run on the host from the pod, it would be great, we could run any command in the pod.